codenvy-api-runner/src/main/java/com/codenvy/api/runner/RunnerService.java

```diff
@@ -63,10 +75,19 @@
 @Path("/runner/{ws-id}")
 @Description("Runner REST API")
 public class RunnerService extends Service {
+    private static final String DOCKERFILES_REPO = "runner.docker.dockerfiles_repo";
```

Comment:
You shoud move this properties to com.codenvy.api.runner.internal.SlaveRunnerService
it becouse script locate on slave runners not on API